core/templates/dev/head/pages/header_js_libs.html

```diff
@@ -1,8 +1,8 @@
 <script src="/assets/rich_text_components_definitions.js"></script>
 <script src="/assets/constants.js"></script>
-{% if not DEV_MODE %}
-  <script src="/assets/hashes.js"></script>
-{% endif %}
+<foobar ng-if="!DEV_MODE">
```

Comment:
is this any kinda directive? (re foobar)
can we use ng-if in script tag?

Reply:
Hi.... I will fix this!